serverless/src/submitSwodlrOrder/handler.js

```diff
@@ -0,0 +1,190 @@
+import 'array-foreach-async'
+import axios from 'axios'
+import { v4 as uuidv4 } from 'uuid'
+
+import { constructOrderPayload } from './constructOrderPayload'
+import { getDbConnection } from '../util/database/getDbConnection'
+import { parseError } from '../../../sharedUtils/parseError'
+import { startOrderStatusUpdateWorkflow } from '../util/startOrderStatusUpdateWorkflow'
+
+const graphQlQuery = `
+mutation GenerateNewL2RasterProduct ($cycle: Int!, $pass: Int!, $scene: Int!, $outputGranuleExtentFlag: Boolean!, $outputSamplingGridType: GridType!, $rasterResolution: Int!, $utmZoneAdjust: Int, $mgrsBandAdjust: Int) { 
+  generateL2RasterProduct(cycle: $cycle, pass: $pass, scene: $scene, outputGranuleExtentFlag: $outputGranuleExtentFlag, outputSamplingGridType: $outputSamplingGridType, rasterResolution: $rasterResolution, utmZoneAdjust: $utmZoneAdjust, mgrsBandAdjust: $mgrsBandAdjust) {
+    cycle
+    pass
+    scene
+    outputGranuleExtentFlag
+    outputSamplingGridType
+    rasterResolution
+    utmZoneAdjust
+    mgrsBandAdjust
+    id
+    timestamp
+    status {
+        id
+        timestamp
+        state
+        reason
+    }
+  }
+}`
+
+/**
+ * Submits an order to CMR Ordering (cmr-ordering)
+ * npm run invoke-local -- --function submitSwodlrOrder -p tmp/test_echo_order.json
```

Comment:
```suggestion
 * npm run invoke-local -- --function submitSwodlrOrder -p tmp/test_swodlr_order.json
```
its just an example we gitignore the tmp folder